README.md

```diff
@@ -54,6 +54,9 @@
 
 Dify is an open-source LLM app development platform. Its intuitive interface combines agentic AI workflow, RAG pipeline, agent capabilities, model management, observability features and more, letting you quickly go from prototype to production. 
 
+### Dify OpenVINO Model Server Integration for local LLM inferencing
```

Comment:
We prefer not to modify this file, you can submit a pull request to https://github.com/langgenius/dify-docs instead.